bundles/org.openhab.ui/web/src/components/widgets/system/oh-sipclient.vue

```diff
@@ -0,0 +1,248 @@
+<template>
+  <div>
+    <!-- Show settings gear for local settings if intercom is enabled and in edit mode -->
+    <f7-button v-if="context.editmode" :style="{ height: config.iconSize + 'px' }" icon-f7="gear_fill" :icon-size="config.iconSize + 'px'" @click.stop="localSettingsPopup()" />
```

Comment:
Does the edit mode icon need to use the configured icon size?  This seems unexpected to me since it feels like more part of the editor and not actually a feature of the runtime widget.

Reply:
No, it does not need to.
It’s one of the design decisions I took when I created the widget but it’s always good to have some discussion. 

When we we are talking about changing the size of the edit mode icon, I would also place this at the right edge because if widget configuration is generally placed over the right edge of a widget.

Reply:
Done, see https://github.com/openhab/openhab-webui/pull/1453/commits/11e3d788829e2f74b2b106bd210d2a07838aef4d